emoji/src/test/java/com/vanniktech/emoji/EmojiManagerTest.java

```diff
@@ -68,4 +82,10 @@
 
     assertThat(EmojiManager.getInstance().findEmoji("")).isNull();
   }
+
+  @Test public void findAllEmojisEmpty() {
```

Comment:
this only tests the case of `findAllEmojis` when it's empty. It should also test the actual logic if it's not empty